storage/client/client.go

```diff
@@ -685,6 +687,49 @@ func (c *StorageClient) Delegations(ctx context.Context, address staking.Address
 	return &ds, nil
 }
 
+// DelegationsTo returns a list of delegations to an address.
+func (c *StorageClient) DelegationsTo(ctx context.Context, address staking.Address, p apiTypes.GetConsensusAccountsAddressDelegationsToParams) (*DelegationList, error) {
+	res, err := c.withTotalCount(
+		ctx,
+		queries.DelegationsTo,
+		address.String(),
+		p.Limit,
+		p.Offset,
+	)
+	if err != nil {
+		return nil, wrapError(err)
+	}
+	defer res.rows.Close()
+
+	ds := DelegationList{
+		Delegations:         []Delegation{},
+		TotalCount:          res.totalCount,
+		IsTotalCountClipped: res.isTotalCountClipped,
+	}
+	for res.rows.Next() {
+		d := Delegation{
+			ValidatorAddress: address.String(),
+		}
+		var shares, escrowBalanceActive, escrowTotalSharesActive common.BigInt
+		if err := res.rows.Scan(
+			&d.Address,
+			&shares,
+			&escrowBalanceActive,
+			&escrowTotalSharesActive,
+		); err != nil {
+			return nil, wrapError(err)
+		}
+		amount := new(big.Int).Mul(&shares.Int, &escrowBalanceActive.Int)
+		amount.Quo(amount, &escrowTotalSharesActive.Int)
+		d.Amount = BigInt{Int: *amount}
```

Comment:
We do this ugly dance in four places now; WDYT about extracting it into a helper func?

Reply:
Good idea, added